test/asn1_internal_test.c

```diff
@@ -40,14 +40,15 @@ static int test_tbl_standard()
         last_nid = tmp->nid;
     }
 
-    if (last_nid != 0) {
-        fprintf(stderr, "asn1 tbl_standard: Table order OK\n");
+    if (TEST_int_ne(last_nid, 0)) {
+        TEST_info("asn1 tbl_standard: Table order OK");
         return 1;
     }
 
+    TEST_error("asn1 tbl_standard: out of order");
     for (tmp = tbl_standard, i = 0; i < OSSL_NELEM(tbl_standard); i++, tmp++)
-        fprintf(stderr, "asn1 tbl_standard: Index %" OSSLzu ", NID %d, Name=%s\n",
-                i, tmp->nid, OBJ_nid2ln(tmp->nid));
+        TEST_note("asn1 tbl_standard: Index %" OSSLzu ", NID %d, Name=%s",
```

Comment:
Switch from fprintf to BIO_printf (implicitly in this case) invalidates OSSLzu. I mean whenever you switch you must replace OSSLzu with zu. That's by BIO_printf design, it provides RTL-independent interface.